test/unit/test_inference_engine_plugins.py

```
@@ -1273,3 +1280,502 @@ def test_get_available_backends_windows(gpu_env: Optional[str], expected_backend
+# _enumerate_store_gguf_models
+# ---------------------------------------------------------------------------
+
+
+def _make_ref_json_file(model_files):
+    """Create a mock RefJSONFile with the given model files."""
+    ref = MagicMock()
+    ref.model_files = model_files
+    return ref
+
+
+def _make_store_file(hash, file_type):
```

Ruff (0.15.15)

[error] Function argument `hash` is shadowing a Python builtin

(A002)